### ISSUE TITLE
Wallet hasn't worked since 8.3

### ISSUE THREAD

**Issue by dcca122** [ethereum/mist#1242]
THIS IS A NOOB POST...

I am not a computer science major or anything, just a casual user for a while now. Since 8.3 my wallet has stopped working/synching. Is there a set up process somewhere that I can work my way through to troubleshoot my synching issue?

#### System information
``````
Version: `0.8.4`
OS & Version: windows/linux/osx
Node type: `eth/geth(default)`
``````

#### Please check the **already existing** issues to keep duplicates at a minimum.
Furthermore several **work-arounds** have been collected in the [Mist-Troubleshooting-Guide](https://github.com/ethereum/wiki/wiki/Mist-Troubleshooting-Guide).

If possible add the following to your report:

- Screenshots

- Check the console, of Mist (`CTRL/CMD + ALT + i`) and take a screenshot

- Log files (Go to `menu -> accounts -> backup -> application data` and upload as zip-archive):
  -   osx: `~/Library/Application Support/Mist/node.log` 
  -   windows: `%APPDATA%/Roaming/Mist/node.log`
  -   linux: `~/.config/Mist/node.log`

**Comment by Doncharly** [comment]
https://kiboplatform.net/DonKibo

https://kiboplatform.net/es/advantages.html

http://huaco2016.wixsite.com/kibo-ingledoncharly

*KIBO. Loto Registracion Only 10 Ethereum Unique Pay*
*Skype Carlos.Silva522*

2016-10-01 18:30 GMT-05:00 dcca122 <notifications@github.com>:

**Comment by oneala** [comment]
i am having the same problem... have you tried with geth directly ? i just did but i get "timed out DAO-fork check, dropping" ... using latest version.. i am stuck!

PLEASE SOMEONE HELP US!

**Comment by frozeman** [comment]
The network was under attack and therefore are sinking is very slow.

Please find the latest mist version or run geth separately. 

 Importing blocks can take up to 30 minutes per few hundered blocks. 

You can also wait for the latest geth version in and tried with this one.

**Comment by oneala** [comment]
But i am using the latest version, also i am running geth separately now.. all i get is this: https://gyazo.com/ef967e9475accb1c18ed05393b08125c

**Comment by dcca122** [comment]
I ended up deleting my chain file and downloading entire chain again. It
ended up working but was taking so long that I ended up just using
Myetherwallet

I was just trying to get rid of some Rep Tokens

**Comment by luclu** [comment]
@dcca122, @oneala, we just published [release `0.8.6`](https://github.com/ethereum/mist/releases/tag/v0.8.6) which will download **geth** `1.4.18` on-the-fly. 
*Please* update to easily roll with the upcoming [Hard Fork](https://blog.ethereum.org/2016/10/13/announcement-imminent-hard-fork-eip150-gas-cost-changes/) on Tuesday.
